bothub/common/tasks.py

```diff
@@ -540,3 +540,17 @@ def get_project_organization(project_uuid: str):  # pragma: no cover
     grpc_client = ConnectGRPCClient()
     authorizations = grpc_client.list_authorizations(project_uuid=project_uuid)
     return authorizations
+
+
+@app.task(name="remove_authorizations_project")
+def remove_authorizations_project(project_uuid: str, authorizations_uuids: list):
+    for authorization_uuid in authorizations_uuids:
+        grpc_client = ConnectGRPCClient()
```

Comment:
You can put the class initialization before the iteration, that way you make the channel reused over and over again.